deployment/README.md

```diff
@@ -1,73 +1,148 @@
+
+
+## Create/Download EC2 KeyPair
 
-After at least one AMI of each type exists, use the following command to generate all of the CloudFormation templates:
+The EC2 KeyPair is used to authenticate SSH connections to the instances from a local machine. Creating this Key is necessary to launch an instance; however, if you already have a Key you can simply re-use that. Note, keys are tied to regions so if you want to launch a stack in a different region, you need to create a key for that region.
+
+## Configure AWS Profile using the AWS CLI
+
+Using the AWS CLI, create an AWS profile.
 
 ```bash
-$ make build
+$ aws configure
 ```
 
-#### Launch the AWS Virtual Private Cloud (VPC)
+You will be prompted to enter your AWS access keys,  AWS access key IDs, and default region. These credentials will be used to authenticate calls to the API when using Boto, Packer, and the AWS CLI.
+
+## Edit Configuration File
 
-Use the following command to create the VPC stack:
+A configuration file is required to launch the stack. An example file is available in this directory at `geotrellis-cluster.cfg.example`.
 
 ```
-$ make vpc-stack
+[CONFIG NAME]
+IPAccess: '<IP to allow SSH Access> (e.g. 216.158.51.82/32)'
+KeyName: '<EC2 Key to authenticate with SSH>'
+MesosFollowerAMI: '<AMI ID of Mesos Follower (optional -- will be found automatically if not provided)>'
+MesosFollowerInstanceProfile: '<ARN of Mesos Follow Instance Profile (optional -- may not be necessary)>'
+MesosFollowerInstanceType: '<Mesos Follow Instance Type>'
+MesosFollowerSpotPrice: '<Spot Price to use Mesos Follower Instances> (optional)>'
+MesosLeaderAMI: '<AMI ID of Mesos Leader (optional -- will be found automatically if not provided)>'
+MesosLeaderInstanceProfile: '<ARN of Mesos Leader Instance Profile (optional -- may not be necessary)>'
+MesosLeaderInstanceType: '<Mesos Leader Instance Type>'
+MesosNotificationARN: ''
+Region: '<AWS Region to Launch Stack>'
+StackType: '<Type of Stack to launch (e.g. accumulo)>'
 ```
 
-#### Create Route 53 Private Hosted Zones
+Multiple configs can be present in the same file, but must be delineated by separate sections using different section headers (`[SECTION]`).
+
+## Launching and Managing Stacks
+
+Creating AMIs and launching stacks is managed with  `gt-stack.py` CLI. This command provides an interface for automatically generating AMIs and launching GeoTrellis Cluster stacks.
 
-Next, create the internal to the VPC private hosted zones:
+### Generate AMIs
+
+Before launching your GeoTrellis Cluster stack you will need to generate AMIs. The `create-ami` subcommand in `gt-stack.py` can be used for generating AMIs.
 
 ```bash
-$ make private-hosted-zones
+usage: gt-stack.py create-ami [-h] [--aws-profile AWS_PROFILE]
```

Comment:
Having these in the `README` is helpful, but potentially a significant maintenance burden as the command line API changes.